documentation.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Get Started | gotham UI</title>
+    <link rel="stylesheet" href="/style.css" />
+  </head>
+  <body>
+    <nav id="nav-bar">
+      <div id="nav-brand">
+        <h1>gotham<span>UI</span></h1>
+      </div>
+      <div id="nav-menu">
+        <div id="nav-home">home</div>
+        <div id="nav-docs">documentation</div>
+      </div>
+    </nav>
+    <div class="container">
+      <aside id="sidebar">
+        <section id="sidebar-get-started" class="sidebar-items">get started</section>
```

Comment:
Better to use ul and li items to list the items on sidebar instead of section.

Reply:
will refactor it